.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn precommit
```

Comment:
if we don't use precommit script, how about deleting it from package.json?

Reply:
good catch! done